scripts/build/bundles.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
+
+const path = require("path");
+
+const parsers = [
+  {
+    input: "src/language-js/parser-babylon.js",
+    target: "universal",
+    minify: true
+  },
+  {
+    input: "src/language-js/parser-flow.js",
+    target: "universal",
+    minify: true,
+    strict: false
+  },
+  {
+    input: "src/language-js/parser-typescript.js",
+    target: "universal",
+    minify: true
+  },
+  {
+    input: "src/language-css/parser-postcss.js",
+    target: "universal",
+    bundler: "webpack",
+    minify: true
+  },
+  {
+    input: "src/language-graphql/parser-graphql.js",
+    target: "universal",
+    minify: true
+  },
+  {
+    input: "src/language-markdown/parser-markdown.js",
+    target: "universal",
+    minify: true
+  },
+  {
+    input: "src/language-vue/parser-vue.js",
+    target: "universal",
+    minify: true
+  },
+  {
+    input: "src/language-handlebars/parser-glimmer.js",
+    target: "node",
+    commonjs: {
+      namedExports: {
+        "node_modules/handlebars/lib/index.js": ["parse"],
+        "node_modules/@glimmer/syntax/dist/modules/es2017/index.js": "default"
+      },
+      ignore: ["source-map"]
+    },
+    transpile: true,
+    minify: true
+  },
+  {
+    input: "src/language-html/parser-parse5.js",
+    target: "node",
+    minify: true
+  }
+].map(parser => {
+  const name = getFileOutput(parser)
+    .replace(/\.js$/, "")
+    .split("-")[1];
+  return Object.assign(parser, { type: "plugin", name });
+});
+
+const bundles = [
```

Comment:
Could we add a JSDoc comment explaining the shape of a bundler and its allowed fields? You have to read through a lot of separate files in the build script right now to understand it. I know @ikatyang knows how to make those kind of JSDoc comments (I don't).

Reply:
FYI, https://github.com/Microsoft/TypeScript/wiki/JSDoc-support-in-JavaScript

Reply:
Thanks